backend/canisters/community/impl/src/lib.rs

```diff
@@ -103,7 +104,7 @@ impl RuntimeState {
             .collect();
 
         let owner_count = owners.len() as u128;
-        let owner_share = (amount_available * 4 / 5) / owner_count;
+        let owner_share = (amount_available * (100 - SNS_FEE_SHARE_PERCENT / 100)) / owner_count;
```

Comment:
Is that going to do `(100 - SNS_FEE_SHARE_PERCENT) / 100` or `100 - (SNS_FEE_SHARE_PERCENT / 100)`?